app/models/mail_server_log/delivery_status.rb

```diff
@@ -0,0 +1,66 @@
+# -*- encoding : utf-8 -*-
+class MailServerLog::DeliveryStatus
+  include Comparable
+
+  # The order of these is important as we use the keys for sorting in #<=>
+  HUMANIZED = {
```

Comment:
I'm going to merge this as is, as it's an existing problem in the MTA-specific classes. We'll catch all of these cases in https://github.com/mysociety/alaveteli/pull/3864